src/web/app/pages-student/student-home-page/student-home-page.component.spec.ts

```diff
@@ -36,4 +36,60 @@ describe('StudentHomePageComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should snap with null fields', () => {
+    expect(fixture).toMatchSnapshot();
+  });
+
+  it('should snap with blank fields', () => {
+    const studentName: string = '';
+    const studentCourse: any = {
+      course: {},
+      feedbackSessions: [],
+    };
+    component.user = studentName;
+    component.courses = [studentCourse];
+    fixture.detectChanges();
+    expect(fixture).toMatchSnapshot();
+  });
+
+  it('should snap with populated fields', () => {
+    const studentName: string = 'John Doe';
+    const studentCourse: any = {
+      course: {
+        id: 'CS2103',
+        name: 'Software Engineering',
+      },
+      feedbackSessions: [
+        {
+          feedbackSession: {
+            feedbackSessionName: 'First Session',
+            courseID: 'CS3281',
```

Comment:
This should be `courseId` (note the lowercase `d`). You can see in the snap that this session is not rendered.
Also, because the `course` object's ID is `CS2103`, maybe better to put everything as `CS2103` instead. Or whatever as long as it is consistent.